### PR TITLE
Reader: make Follow/Following button more buttony

### PR DESCRIPTION
**Pull request by jancavan**
This addresses: https://github.com/Automattic/wp-calypso/issues/3121

**Before:**
screenshot 2016-05-05 19 48 39

screenshot 2016-05-11 14 05 45

**After:** 
Changed in several different places: https://github.com/Automattic/wp-calypso/pull/5256#issuecomment-218915631

### CODE REVIEW

**Comment by lancewillett** [comment]
Sorry about the bot labeling: I'm tweaking Waffle.io settings and it might be a bit *too* hungry to change things. :)

**Comment by blowery** [comment]
@jancavan do we want to try to a/b test this?

**Comment by blowery** [comment]
@jancavan what's the followed state look like with this change?

**Comment by jancavan** [comment]
**In Calypso:**
------------

**Individual site stream:**

Before:
screenshot 2016-05-12 15 44 11

screenshot 2016-05-12 15 44 51

After:
screenshot 2016-05-12 14 50 56
screenshot 2016-05-12 15 45 10

**Card popover:**

Before:
screenshot 2016-05-12 15 47 19

screenshot 2016-05-12 15 47 12

After:
screenshot 2016-05-12 15 46 18

screenshot 2016-05-12 15 46 24

**Recommendations:**

Before:
screenshot 2016-05-12 15 48 50
screenshot 2016-05-12 15 48 57

After:
screenshot 2016-05-12 15 48 25
screenshot 2016-05-12 15 49 13

**Manage Follows (Search):**

Before:
screenshot 2016-05-12 15 50 58

After:
screenshot 2016-05-12 15 51 10

**Manage Follows (Following list):**

Before:
screenshot 2016-05-12 15 51 36

After:
screenshot 2016-05-12 15 51 20

**Card header:**

Before:
screenshot 2016-05-12 15 55 53
screenshot 2016-05-12 15 55 49

After:
screenshot 2016-05-12 15 56 03
screenshot 2016-05-12 15 55 59


**Full-post view:**

Before:
screenshot 2016-05-12 16 02 22
screenshot 2016-05-12 16 02 20

After:
screenshot 2016-05-12 16 02 04
screenshot 2016-05-12 16 02 07

**Tags:**

Before:
screenshot 2016-05-12 16 03 22
screenshot 2016-05-12 16 03 26

After:
screenshot 2016-05-12 16 03 31
screenshot 2016-05-12 16 03 49

**Lists:**

Before:
screenshot 2016-05-12 16 07 08
screenshot 2016-05-12 16 07 13

After:
screenshot 2016-05-12 16 07 40
screenshot 2016-05-12 16 07 35

In WPCOM:
----------

This has not been deployed yet, but I've sent a patch for review:

**Actionbar:**

Before:
screenshot 2016-05-12 16 31 51
screenshot 2016-05-12 16 31 42

After:
screenshot 2016-05-12 16 30 15
screenshot 2016-05-12 16 30 19

**Actionbar (folded):**

Before:


After:


**Logged out subscription box:**

Before:
screenshot 2016-05-12 13 30 37

After:
screenshot 2016-05-12 16 12 37

I think the we can just roll this out and create a different PR for the abtest. Thoughts?

**Comment by blowery** [comment]
Looks good! The font color in the Action Bar seems off, but otherwise perfect.

> I think the we can just roll this out and create a different PR for the abtest. Thoughts?

Agreed.

**Comment by blowery** [comment]
Tested, looks good!

File: client/reader/post-options/_style.scss [hunk 1]
```diff
@@ -54,7 +54,24 @@
 	}
 }
 
+.popover__menu {
+
+	.gridicon {
```
Comment (gwwar) [inline client/reader/post-options/_style.scss]: This selector is not specific enough to the reader and is being picked up by other popover menus, like in /pages. I suggest adding a component class name to the reader popover menu.

screen shot 2016-05-13 at 4 15 34 pm

**Pull request merged**